### DISCUSSION TITLE
Is there any paper or documentation about model Architecture of XTTS and XTTS_v2 models?

### DISCUSSION THREAD

**Discussion by chazo1994** [coqui-ai/TTS#3668] [General Q&A]
Hi Everyone, I want to research about XTTS and XTTS_v2 model. But I cannot find the paper or documentation about the detailed architecture, experiments of XTTS and XTTS_v2 model. Therefore, it would be great if anyone would share these documents with me.

**Comment by yalsaffar** [comment]
Hey! I looked around for that specific paper but no luck finding anything. However, I did come across the release notes for XTTS v2 on Erogol's site, which are pretty insightful. 
They might not be exactly what you were after, but definitely worth checking out. You can find them here: 
[XTTS v2 Release and Notes](https://erogol.com/2023/11/11/xtts-v2-release-and-notes)
Hope this helps!
